app/build.gradle

```diff
@@ -36,6 +36,7 @@ android {
     }
     buildTypes {
         debug {
+            pseudoLocalesEnabled true
```

Comment:
This indicates that our app supports pseudolocales (https://developer.android.com/guide/topics/resources/pseudolocales.html). We can leave this enabled, and it won't affect anything day to day; not unless we're doing locale-related work and meet the following criteria.

Pseuodolocales will show if all of the following criteria are met:

1. app is in `DEBUG` mode
2. developer settings are enabled
3. user has selected a pseuolocale from their device language settings

https://developer.android.com/guide/topics/resources/pseudolocales.html